### DISCUSSION TITLE
Make TabBarIOS scroll up with the page content

### DISCUSSION THREAD

**Discussion by zsoltbokor** [react-native-tvos/react-native-tvos#333] [Feature Requests]
### Description

Hello, we are working on a project for tvOS and one of the requirement is to use the tab bar but while the user is pressing DOWN scroll up the tab bar together with the content and vice versa. I was trying to put the tab bar in a ScrollView but then the content of the tab was not rendered. Can you please confirm if this behaviour can be achieved using that component or we need to create a custom one. Thank you!

### Version

68

### Output of `npx react-native info`

-

### Steps to reproduce

-

**Comment by douglowder** [comment]
You will need to create a custom component for this. `TabBarIOS` maps directly to the native `UITabBar`, which works differently on tvOS and does not allow customization. See https://developer.apple.com/documentation/uikit/uitabbarcontroller?language=objc and the section titled "Differences Between iOS and tvOS".

**Comment by istvan-szilagyi** [comment]
Actually it has 2 behaviours without the need of customisation. By default as you said it's sticky, but if the content of the tab is scrollable, if you set [tabBarObservedScrollView](https://developer.apple.com/documentation/uikit/uiviewcontroller/3152928-tabbarobservedscrollview?language=objc) to the view controller that controls a tab, the tab bar will be scrolling with the content. Do you know if this property is reachable through react native?

**Comment by douglowder** [comment]
This property is not currently reachable from React Native. I'll reopen this and change it to a feature request, and see what effort is involved.
